README.rst

```diff
@@ -31,11 +31,21 @@ This will start up a Flask app, and will print out a line such as::
 
 You can then call pip with::
 
-   pip install --index-url http://127.0.0.1:5000/ astropy
+   pip install --index-url http://127.0.0.1:5000/simple/ astropy
```

Comment:
Also caused by some inflexibility on `simple-repository-server` (at https://github.com/simple-repository/simple-repository-server/blob/main/simple_repository_server/routers/simple.py#L41).

There is no good reason for this limitation, but it would require some more thought than I have capacity for this evening :sleeping: